widgets/filters/src/FilterActions.ts

```diff
@@ -0,0 +1,11 @@
+export default class FilterActions {
+    toolbox: Stage.Types.Toolbox;
+
+    constructor(toolbox: Stage.Types.Toolbox) {
+        this.toolbox = toolbox;
+    }
```

Comment:
Optional:

Here's a cool trick to consider :slightly_smiling_face: 

```suggestion
    constructor(private toolbox: Stage.Types.Toolbox) {}
```

Although, `no-useless-constructor` is not too happy about it. We would need to disable it and enable https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-useless-constructor.md to fix the false-positive. I guess this should be done in `cloudify-ui-common` at some point

If you decide to apply the suggestion, I'm ok with having a `eslint-disable-next-line` or changing the ESLint config file and adding the `@typescript-eslint` version there :slightly_smiling_face:

Reply:
I applied the suggestion and used `eslint-disable-next-line` for now.